internal/dependencymanager/dependencyinstaller.go

```diff
@@ -74,6 +76,11 @@ type dependencyManagerFlagsCollection struct {
 	skipAlias       bool `default:"false" flag:"skip-alias" info:"Skip prompting for an alias"`
 }
 
+func (f *dependencyManagerFlagsCollection) AddToCommand(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&f.skipDeployments, "skip-deployments", false, "Skip adding the dependency to deployments")
+	cmd.Flags().BoolVar(&f.skipAlias, "skip-alias", false, "Skip prompting for an alias")
```

Comment:
Option no. 2 is to continue using sconfig to bind the flags & then the syntax could remain unchanged

```suggestion
err := sconfig.New(f).
	FromEnvironment(util.EnvPrefix).
	BindFlags(cmd.Flags()).
	Parse()
```